server/src/main/resources/db/changelog/20201103112757-purge-current-sca-certs.xml

```diff
@@ -7,21 +7,38 @@
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet id="20201103112757-1" author="crog">
+        <validCheckSum>7:b060e85fd3270acbb9778b17c6d0a447</validCheckSum>
         <comment>
             Removes any existing SCA certs to address a problem with cert generation in some organizations
         </comment>
 
         <sql>
             UPDATE cp_cert_serial cs SET revoked = true
-                WHERE EXISTS (SELECT cac.serial_id FROM cp_cont_access_cert cac WHERE cac.serial_id = cs.id);
+            WHERE EXISTS (SELECT cac.serial_id FROM cp_cont_access_cert cac WHERE cac.serial_id = cs.id);
```

Comment:
Keep that indentation, as it's a continuation of the previous statement.

Reply:
Done.